packages/pubsub/system-test/pubsub.js

```diff
@@ -144,11 +139,9 @@ describe('pubsub', function() {
     it('should allow manual paging', function(done) {
       pubsub.getTopics({
         pageSize: TOPIC_NAMES.length - 1
-      }, function(err, topics, nextQuery) {
+      }, function(err, topics) {
```

Comment:
I guess I'm just lost why we need to be manually handling pagination options at this level. Initially I assumed that the `paginator` object would handle cleaning up options like that, but it does not.

I would prefer that we either have the pagintor class clean up autoPaginate values.
Or we just rename gaxOpts to something else (requestConfig, etc.) and document the basic level options.